spec/features/post_spec.rb

```diff
@@ -1,9 +1,10 @@
 require 'rails_helper'
 
-RSpec.describe 'Posts', type: :feature, feat: true do
```

Comment:
- [ ] Good job so far 👏 But some features especially for posts are missing 😢  As the previous reviewer said Please make sure that all featured tests are implemented and include the all tests to meet all the project requirements  👍

Reply:
Hey 
@codecaiine Ive have just added them